query/src/sql/parsers/parser_user.rs

```diff
@@ -37,59 +35,18 @@ use crate::sql::statements::DfGrantRoleStatement;
 use crate::sql::statements::DfRevokePrivilegeStatement;
 use crate::sql::statements::DfRevokeRoleStatement;
 use crate::sql::statements::DfShowGrants;
-use crate::sql::statements::DfUserWithOption;
 use crate::sql::DfParser;
 use crate::sql::DfStatement;
 
 impl<'a> DfParser<'a> {
     pub(crate) fn parse_create_user(&mut self) -> Result<DfStatement<'a>, ParserError> {
-        let if_not_exists =
-            self.parser
-                .parse_keywords(&[Keyword::IF, Keyword::NOT, Keyword::EXISTS]);
-        let (username, hostname) = self.parse_principal_name_and_host()?;
-        let with_options = self.parse_user_options()?;
-        let auth_option = self.parse_auth_option()?;
-
-        let create = DfCreateUser {
-            if_not_exists,
-            user: UserIdentity { username, hostname },
-            auth_option,
-            with_options,
-        };
-        Ok(DfStatement::CreateUser(create))
+        // skip to the planner v2 syntax
+        self.skip_all(DfStatement::CreateUser(DfCreateUser::default()))
```

Comment:
I prefer to convert `DfStatement::CreateUser` to a unit like `DfStatement::Copy` and remove `DfCreateUser` directly (so does `parse_create_user`).

So that we don't need `Default` on `DfCreateUser` anymore.

Reply:
We can wait for https://github.com/datafuselabs/databend/pull/6895

So that we don't need those trick.